libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -259,9 +259,7 @@ float AP_MotorsMulticopter::get_current_limit_max_throttle()
     }
 
     // calculate the maximum current to prevent voltage sag below _batt_voltage_min
-    float batt_current_max = MIN(_batt_current_max, _batt_current + (_batt_voltage-_batt_voltage_min)/_batt_resistance);
```

Comment:
Removing this doesn't look correct to me, but Randy and Leonard know much more about it than me and will give their review. If it is correct though, the comment above it needs to be removed too.